templates/templates/kyma-azure-event-hubs-namespaces-cleanup.yaml

```diff
@@ -0,0 +1,71 @@
+test_infra_ref: &test_infra_ref
+  org: kyma-project
+  repo: test-infra
+  path_alias: github.com/kyma-project/test-infra
+
+kyma_ref: &kyma_ref
+  org: kyma-project
+  repo: kyma
+  path_alias: github.com/kyma-project/kyma
+
+base_image_116: &base_image_116
+  image: eu.gcr.io/kyma-project/test-infra/kyma-integration:v20200817-1.14.0-rc1-75-g625c1054-k8s1.16
+
+azure_event_hubs_namespaces_cleanup_job_template: &azure_event_hubs_namespaces_cleanup_job_template
+  decorate: true
+  path_alias: github.com/kyma-project/kyma
+  max_concurrency: 1
+  spec:
+    containers:
+      - <<: *base_image_116
+        securityContext:
+          privileged: true
+        command:
+          - "bash"
+        args:
+          - "-c"
+          - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/cluster-integration/helpers/cleanup-azure-event-hubs-namespaces.sh"
+        env:
+          - name: AZURE_SUBSCRIPTION_APP_ID
```

Comment:
There is already a [preset](https://github.com/kyma-project/test-infra/blob/28bdc7b7b9d0069dcba4455aa4462bc803b071e8/prow/config.yaml#L197) meant for this purpose.

One example of using this is [here](https://github.com/kyma-project/test-infra/blob/master/prow/jobs/kyma/kyma-upgrade-gardener.yaml#L17).

Reply:
Done.